services/api/kalvi/settings.py

```diff
@@ -159,3 +159,11 @@
 }
 
 PASSWORD_RESET_TIMEOUT=900     #900 Sec = 15 Min
+
+# Email Configuration
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'smtp.gmail.com'
```

Comment:
Grab the value from the environment variable. Through Docker compose. Refer how its done for DB setup

https://github.com/kalvilabs/kalvios/blob/master/services/api/kalvi/settings.py#L84

Reply:
yes i had done that